gdal/ogr/ogrsf_frmts/flatgeobuf/ogr_flatgeobuf.h

```diff
@@ -42,6 +42,9 @@ class OGRFlatGeobufDataset;
 
 static constexpr const uint8_t magicbytes[8] = { 0x66, 0x67, 0x62, 0x00, 0x66, 0x67, 0x62, 0x00 };
 
+static constexpr const uint32_t header_max_buffer_size = 1048576;
+static constexpr const uint32_t feature_max_buffer_size = 1073741824 - 1;
```

Comment:
The `const` in `constexpr const` is redundant.

It would be needed only if it was something like constexpr reference to a constant object i.e. `static constexpr int const&`.

Reply:
Any particular reason for the  - 1 ?

Reply:
2GB - 1 is the hard limit for a single flatbuffer, see https://github.com/google/flatbuffers/blob/917687c7a6357d4a0e9e53e0f051b76cd0187747/include/flatbuffers/base.h#L298-L299.